pkg/server/router/webhook.go

```diff
@@ -229,10 +229,11 @@ func (router *router) handleGitlabWebhook(request *restful.Request, response *re
 		}
 
 		performParams = &api.PipelinePerformParams{
-			Ref:         fmt.Sprintf(gitlabMergeRefTemplate, objectAttributes.ID),
+			Ref:         fmt.Sprintf(gitlabMergeRefTemplate, objectAttributes.Iid),
```

Comment:
Also need to update https://github.com/caicloud/cyclone/pull/460/files?diff=split#diff-0bcfcc4b4602815fc3bec609a40c7c7bR256

Reply:
done